FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -148,7 +148,7 @@ class ComposeViewController: BaseViewController {
             : txtSubject.text ?? "(no subject)"
 
         let from = GoogleApi.instance.getEmail()
-        let replyToMimeMsg = replyToMime.map { String(data: $0, encoding: .utf8) }
+        let replyToMimeMsg = replyToMime.flatMap { String(data: $0, encoding: .utf8) }
```

Comment:
Curious why the flatMap here?

Reply:
because `String(data: $0, encoding: .utf8)` will return you optional `Data?` in case it would be replyToMime.map {   }, your replyToMimeMsg will be: `String??` 
flatMap will flattened failable init for you

Reply:
Oh so flatMap gets rid of optionals?